src/content/community/get-involved/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How can I get involved?
+description: How to get involved in the Ethereum community.
+sidebar: true
+lang: en
+---
+
+# How can I get involved? {#get-involved}
+
+The Ethereum community includes people of many different backgrounds and skillsets. Whether you’re a developer, an artist, or an accountant, there are ways to get involved. Here’s a list of suggestions that might help you get started.
+
+## Developers 💻 {#developers}
+
+- Learn about and try Ethereum at [ethereum.org/developers/](/developers/)
+- [Find a bounty on Gitcoin](https://gitcoin.co/), work on a small or large technical issue, earn crypto!
+- Attend an [ETHGlobal](http://ethglobal.co/) hackathon near you!
+- Check out [projects related to your area of expertise or programming language of choice](/developers/docs/programming-languages/)
+- Watch or participate in the [Core Dev calls](https://www.youtube.com/playlist?list=PLaM7G4Llrb7zfMXCZVEXEABT8OSnd4-7w)
+- [Ecosystem Support Program's wishlist](https://esp.ethereum.foundation/wishlist/) - tooling, documentation, and infrastructure areas where the Ethereum Ecosystem Support Program is actively seeking grant applications
+- [Web3Bridge](https://www.web3bridge.com/) - join the aspiring web3 community in their initiative to identify, train, and support hundreds of developers and community members throughout Africa
+
+## Researchers & Academics <Emoji text=":mag:" size={1} />‍ {#researchers-and-academics}
```

Comment:
These headers with emojis seem to have a weird side effect on the side panel 
![Screen Shot 2021-12-06 at 2 46 26 PM](https://user-images.githubusercontent.com/15589226/144927839-94a169a3-624d-4a1b-894c-90ab9976a887.png)

Reply:
[Fixed in this commit](https://github.com/ethereum/ethereum-org-website/pull/4521/commits/ea7a1c7b6f03f08677f6c7795a772b1cf1aa1825) but doesn't seem very elegant if anyone wants to refactor :-)